packages/data-point/examples/async-example.js

```diff
@@ -1,7 +1,13 @@
-const dataPoint = require('../').create()
+const assert = require('assert')
+const mocks = require('./async-example.mocks')
+
+const DataPoint = require('../')
```

Comment:
does `const DataPoint = require('data-point')` work? that would make the examples look closer to how you actually use the library

Reply:
nope u cant require a dependency within itself, what we could do as a follow up is move examples into its  lerna sub-project, it wouldnt have to be published, or it could...  regardless might be a way to do that what you suggest